wgsl/index.bs

```diff
@@ -6668,6 +6668,109 @@ When an identifier is used as a [=syntax/callable=] item, it is one of:
     | [=syntax/bitwise_expression=]
 </div>
 
+## Operator Precedence and Associativity ## {#operator-precedence-associativity}
+
+This entire subsection is non-normative.
+
+Operator precedence and associativity in right-hand side WGSL expressions
```

Comment:
It feels like we should define precedence and associativity. Also why are RHS expressions called out?

Reply:
I've tried to explain emergent precedence & associativity before the table as that's where they chiefly matter as properties observed from the grammar. I'd appreciate any feedback!

RHS expressions are called out because LHS expressions are very restricted to `*` and `&`. Even the new linking makes it very visible, no subtree from the syntax/expression contains LHS elements.

I think for WGSL v1, it is better to use restricted wording to avoid misreadings.

Thank you!